.github/workflows/main.yml

```diff
@@ -23,14 +23,15 @@ jobs:
           node-version: ${{ matrix.node-version }}
 
       - name: Install dependencies
-        run: yarn
+        run: yarn && yarn bootstrap
 
-      - name: Build Code
+      - name: Build code
         run: yarn build
-        env:
-          CI: true
 
-      - name: Test Code
-        run: yarn test
-        env:
-          CI: true
+      - name: Unit tests
+        run: yarn test --scope electron-redux --stream
+
+      - name: E2E tests
+        uses: GabrielBB/xvfb-action@v1.2
```

Comment:
sure we need selenium? I would've thought headless chrome would've been enough

Reply:
This is not selenium but a virtual display, as far as I'm concerned.

I was referring to this guide: [Testing on Headless CI Systems](https://www.electronjs.org/docs/tutorial/testing-on-headless-ci).

I'll double-check.